isamples_metadata/taxonomy/metadata_models.py

```diff
@@ -55,55 +54,51 @@ def load_model_from_path(collection, label_type):
         elif collection == "OPENCONTEXT" and label_type == "sample":
             MetadataModelLoader._OPENCONTEXT_SAMPLE_MODEL = model
 
-    @staticmethod
-    def initialize_models():
-        """
-            Invokes the load_model function to load all of the possible models
-            that are available based on the config
-        """
-        MetadataModelLoader.load_model_from_path("SESAR", "material")
-        MetadataModelLoader.load_model_from_path("OPENCONTEXT", "material")
```

Comment:
This was defined as a method to be called from `isb_web/main.py` `on_startup()` to instantiate all the metadata models. But it seems that a better way would be instantiating the necessary models that we need through `load_model_from_path()` after checking if the model exists through `get_xxx_model()`. This updated is based on the review, and the fact that not all isamples_boxes will have all the metadata models loaded. We expect that the metadata model that will be used within that box will only be loaded.